### PR TITLE
Let 30% of the validator broadcast cx-receipts

### PR DESCRIPTION
**Pull request by rlan35**
desides, delay the cross-link function for later, and print consensus message on stdout.

### CODE REVIEW

**Comment by rlan35** [comment]
Jenkins, test this please.

File: consensus/consensus_v2.go [hunk 1]
```diff
@@ -792,11 +792,13 @@ func (consensus *Consensus) finalizeCommits() {
...
 	consensus.getLogger().Info().
-		Uint64("blockNum", beforeCatchupNum).
-		Uint64("ViewId", beforeCatchupViewID).
+		Uint64("blockNum", block.NumberU64()).
+		Uint64("ViewId", block.Header().ViewID().Uint64()).
 		Str("blockHash", block.Hash().String()).
 		Int("index", consensus.getIndexOfPubKey(consensus.PubKey)).
 		Msg("HOORAY!!!!!!! CONSENSUS REACHED!!!!!!!")
+	// Print to normal log too
+	utils.GetLogInstance().Info("HOORAY!!!!!!! CONSENSUS REACHED!!!!!!!", "BlockNum", block.NumberU64())
```
Comment (LeoHChen) [inline consensus/consensus_v2.go]: why? we are phasing out the original logs.  will be no more original logs.

Comment (rlan35) [reply consensus/consensus_v2.go]: We still have those “trie commit to database” stuff on the terminal, which is confusing people. It’s etheruem code so hard to remove. Remember a lots of people asking whether I joined consensus or not, right? . We might as well surface up consensus signal just for people to easily see.

Comment (rlan35) [reply consensus/consensus_v2.go]: Just merged the code so other part get in master for release. Let me know if the log code has to be reverted.

**Pull request merged**